pkg/clients/imagebuilder/client.go

```diff
@@ -47,7 +48,19 @@ type OSTree struct {
 
 // Customizations is made of the packages that are baked into an image
 type Customizations struct {
-	Packages *[]string `json:"packages"`
+	Packages            *[]string     `json:"packages"`
+	PayloadRepositories *[]Repository `json:"payload_repositories,omitempty"`
+}
+
+// Repository is the record of Third Party Repository
+type Repository struct {
+	Baseurl    []string `json:"baseurl,omitempty"`
+	CheckGpg   *bool    `json:"check_gpg,omitempty"`
+	GpgKey     *string  `json:"gpg_key,omitempty"`
+	IgnoreSsl  *bool    `json:"ignore_ssl,omitempty"`
+	Metalink   *string  `json:"metalink,omitempty"`
+	Mirrorlist *string  `json:"mirrorlist,omitempty"`
+	Rhsm       bool     `json:"rhsm"`
```

Comment:
Let's stick with CamelCase

```suggestion
	BaseURL  []string `json:"base_url,omitempty"`
	CheckGPG   *bool    `json:"check_gpg,omitempty"`
	GPGKey     *string  `json:"gpg_key,omitempty"`
	IgnoreSSL  *bool    `json:"ignore_ssl,omitempty"`
	Metalink   *string  `json:"metalink,omitempty"`
	Mirrorlist *string  `json:"mirrorlist,omitempty"`
	RHSM       bool     `json:"rhsm"`
```

Reply:
This is for image builder's request so depends on how @ldjebran implements there.

Reply:
I do like with the underscore better for what's worth.

Reply:
yeah for imagebuilder we need to keep base_url as baseurl here for example.

Reply:
I fixed my suggestion for `baseurl` :)
@biancarosa @Saurabh-Thakre I still think we should stick with CamelCase on edge-api variable names. I don't see where it's related to image builder's request as those are internal used variables.